node/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM jmfirth/webpack:7-slim
+
+ENV HOME=/workspace
+
+# COPY package.json npm-shrinkwrap.json $HOME/chat/
+COPY ./package.json $HOME/
+COPY ./docker-up-image.sh /root
+WORKDIR $HOME/
```

Comment:
WORKDIR will create directory if it have not existed before. And it should be placed before the first use.